Source/Core/Core/TitleDatabase.cpp

```diff
@@ -91,8 +94,32 @@ TitleDatabase::TitleDatabase()
 TitleDatabase::~TitleDatabase() = default;
 
 const std::string& TitleDatabase::GetTitleName(const std::string& gametdb_id,
+                                               const std::string& tri_id,
                                                DiscIO::Language language) const
 {
+  if (tri_id != "" || NULL)
```

Comment:
If you want this code to not run, please use `ifdef`, or use `false` instead of `NULL`.

Reply:
not as far as i can tell, no.

fixed.